### ISSUE TITLE
Role & ARIA-label persist on carousel after 'unslicking'

### ISSUE THREAD

**Issue by SirDaev** [Accessible360/accessible-slick#93]
When an accessible-slick slider becomes 'unslicked' (via the responsive settings or via jQuery), the `role` and `aria-label` that are appended upon 'slicking' persist. This makes for excess attributes that a screen reader will read out when the carousel no longer exists. The `role` and `aria-label` should be removed upon destroying the slider.

Minimal demo showing persisting attributes:
https://codepen.io/SirDaev/pen/KKrdoPz

**Before** (when carousel is initialized):
![Screen Shot 2023-06-15 at 10 58 06 AM](https://github.com/Accessible360/accessible-slick/assets/12941979/39438bca-9df3-46d4-90e5-594aae56a3e2)

**After** (when carousel is destroyed via unslick):
![Screen Shot 2023-06-15 at 10 57 22 AM](https://github.com/Accessible360/accessible-slick/assets/12941979/99c35c5a-33b4-41cb-a40d-6df507521b73)
